pkg/api/pod/util.go

```diff
@@ -553,7 +577,7 @@ func subpathInUse(podSpec *api.PodSpec) bool {
 	}
 
 	var inUse bool
-	VisitContainers(podSpec, func(c *api.Container) bool {
+	VisitContainers(podSpec, DefaultContainers, func(c *api.Container, containerType ContainerType) bool {
```

Comment:
consider all persisted containers, not just feature-enabled ones, when determining if this is in use (applies to all `___InUse` calls to VisitContainers)